### ISSUE TITLE
Use yaml front matter to generate idea registry

### ISSUE THREAD

**Issue by oskarth** [status-im/ideas#160]
# Problem

We currently have to update both idea doc and README. This is error-prone and time-consuming.

## Solution

Flip side of https://github.com/status-im/ideas/issues/157 but for idea registry

**Comment by martinklepsch** [comment]
Since this is a bounty it should specify the desired format.

**Comment by oskarth** [comment]
Good point. What do you think is a desirable format that is reasonable to do?

**Comment by martinklepsch** [comment]
I meant the format of the YAML front matter but later saw that you also opened #161. That said one thing that would be a nice intermediate layer is creating a big JSON file from all the front matter. This could then be used for the idea and commitment registries as well as other stuff that we might want to build later on.

**Comment by status-open-bounty** [comment]
![Contract deploying](https://openbounty.status.im/img/deploying_contract.png)
Transaction: [0x9aaf1bd4f8ab542101946a67c4b65cd8d9449d6472ac97850002b41f4fec61e1](https://etherscan.io/tx/0x9aaf1bd4f8ab542101946a67c4b65cd8d9449d6472ac97850002b41f4fec61e1)
